src/Templates/content/TemplatePackagingProject/.template.config/template.json

```diff
@@ -0,0 +1,24 @@
+{
+  "author": "Adam Friedman (tintoy@tintoy.io)",
```

Comment:
(if you want to change, feel free to open another PR or just tell me and I'll update it)